lomap/classes/interval.py

```diff
@@ -304,7 +305,7 @@ def __cmp__(self, other):
             raise TypeError("Unsupported operand types for comparison: '{}' and '{}'."
                     .format(self.__class__.__name__, other.__class__.__name__))
 
-    def __nonzero__(self):
+    def __bool__(self):
```

Comment:
This needs to be tested to ensure semantic equivalence and backwards compatibility.